tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test for vagrant and packaged addons
+# Maintainer: dancermak <dcermak@suse.com>
+
+use strict;
+use warnings;
+use base "basetest";
+use testapi;
+use utils;
+use File::Basename;
+
+
+sub run() {
+    select_console('root-console');
+    zypper_call('in ansible');
+
+    select_console('user-console');
+
+    # version = Tumbleweed, Leap 15 etc
+    my $version = get_required_var('VERSION');
+    my $arch    = get_required_var('ARCH');
+    # FIXME: don't hardcode the build in production
+    # my $build = get_required_var('BUILD');
+    my $build = "20190721";
```

Comment:
this means that the tests would fail again in some days when according older snapshots are not available anymore?
Also please don't keep any FIXME in code. Tickets should be used or a WIP pull request or so

Reply:
This is just a reminder for me to drop that, as I cannot otherwise test this case on o3 with `openqa-clone-custom-git-refspec`. I'll drop this once no testing is required.

Reply:
Why not just overwrite the variable on trigger time? What should work for testing is e.g. `openqa-clone-custom-git-refspec $pr_url $job_id BUILD=20190721`

Reply:
Because I didn't know that was possible.